aptible-auth.gemspec

```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^spec/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'aptible-billing', '~> 1.0'
```

Comment:
Something seems funky with this diff; this has already been removed. Can you rebase?

Reply:
Correct, hadn't rebased yet. This is just straight up the original code. Rebase is done